errors.go

```diff
@@ -89,6 +89,10 @@ const (
 	ErrOffsetsLoadInProgress           KError = 14
 	ErrConsumerCoordinatorNotAvailable KError = 15
 	ErrNotCoordinatorForConsumer       KError = 16
+	ErrInvalidTopic                    KError = 17
```

Comment:
Invalid topics are e.g. blank, or contain invalid characters (presumably `/` and others). Looks like this might finally have fixed https://github.com/Shopify/sarama/blob/351e68b0267f6421505349526c32c8cd259acf53/client.go#L518-L525